guides/common/modules/snip_deprecated-feature.adoc

```diff
@@ -0,0 +1,18 @@
+// When including this file, ensure that attributes set immediately before
+// the include. Otherwise it might result in an incorrect replacement.
+
+[IMPORTANT]
+====
+The {FeatureName} is a deprecated feature.
+Deprecated functionality is still included in {Project} and continues to be supported.
```

Comment:
```suggestion
Deprecated functionality is still included in {Project} {ProjectVersion} and continues to be supported.
```

suggestion (non-blocking): Maybe add the version to make it clear that it's still present in the current version?

Reply:
IDK. @apinnick WDYT?

Reply:
I think it's better not to include a version number unless it will be there permanently (example: "... supported by versions 6.15 and later"). 

Re: current version number. I think it is unnecessary because everything in the guide is supposed to pertain to the current version.

Reply:
That's a good argument -> Let's discard my suggestion.